react-validated-input-component/dist/styles.css

```diff
@@ -0,0 +1,31 @@
+label {
+  font-family: 'Trebuchet MS', sans-serif;
+  font-size: 1rem;
+}
+
+p {
+  font-family: 'Trebuchet MS', sans-serif;
+  font-size: 0.8rem;
+  margin-top: 0.3rem;
+}
+
+input {
+  height: 1.2rem;
+  width: 10rem;
+  border: 2px black solid;
+  margin-top: 0.2rem;
+}
+
+i {
+  margin-left: 1rem;
+  font-size: 1.5rem;
+  vertical-align: -18%;
+}
+
+i.fa-times {
+  color: red;
+}
+
+i.fa-check {
+  color: green;
+}
```

Comment:
In general, we avoid styling classes from libraries. Instead, make your own error and success classes with these colors.